acrg/utils.py

```diff
@@ -140,4 +140,22 @@ def combine_method(resolved, method):
     # apply the combine method to each slice of the grouped data
     output = grouped_data.map(combine_method, method=method)
     
-    return output
\ No newline at end of file
+    return output
+
+def load_json(filename):
+    """Returns a dictionary deserialised from JSON.
+
+    Args:
+        filename: Name of JSON file
+        internal_data: Whether to use data internal to OpenGHG. This refers
```

Comment:
Looks like this part of the docstring can be deleted (looks like this was copied from openghg but isn't relevent here?)